app/views/trips/show.html.erb

```diff
@@ -5,13 +5,26 @@
 <div class="container align-items-center justify-content-center">
   <div class="d-flex flex-column align-items-center justify-content-center">
     <h2>Here are the details of your trip</h2>
+
     <ul>
       <li>Arrival date:
         <%= @trip.arrival_date %>
       </li>
       <li>Departure date:
         <%= @trip.departure_date %>
       </li>
+
+
+<li>
+      <% arrival_start = @trip.arrival_date %>
+      <% departure_start = @trip.departure_date %>
+      <% days_in_range = (arrival_start .. departure_start).count %>
+
+      <% bodie_price = @trip.body.cost_per_day %>
```

Comment:
body_price instead of bodie_price

Reply:
oh yeah sorry ... ill gonna change it ..